jobs/ingestor_s3/templates/config/input_and_output.conf.erb

```diff
@@ -0,0 +1,83 @@
+input
+{
+  s3 {
+    <% if_p("logstash_ingestor.s3.bucket") do |bucket| %>
+      bucket => "<%= bucket %>"
+    <% end %>
+    <% if_p("logstash_ingestor.s3.region") do |region_zone| %>
+    region => "<%= region_zone %>"
+    <% end %>
+    codec => "plain"
+  }
+}
+
+filter
```

Comment:
This filter is specific to audit event logs, right? I think it should go in `src/cf-logstash-filters/src/logstash-filters/audit_event.conf`, then be included in job logstash configuration like so:

https://github.com/cloud-gov/opensearch-boshrelease/blob/1cb69b7bc9e83ee778ed6198538905de4c39320a/jobs/ingestor_syslog/templates/bin/ingestor_syslog#L78-L87

That way, the filter configuration for the job can be dynamic